test/test_gen_backend_stubs.py

```diff
@@ -2,13 +2,22 @@
 
 import os
 import tempfile
+import importlib
 
 from torch.testing._internal.common_utils import TestCase, run_tests
-import tools.codegen.gen_backend_stubs
 
 path = os.path.dirname(os.path.realpath(__file__))
 gen_backend_stubs_path = os.path.join(path, '../tools/codegen/gen_backend_stubs.py')
 
+# This is a hack to easy make the tools subdirectly accessable to this test in CI.
+def import_codegen(module_name: str, module_path: str):
+    spec = importlib.util.spec_from_file_location(module_name, os.path.join(os.path.dirname(path), module_path))
+    module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(module)
+    return module
+
+gen_backend_stubs = import_codegen('gen_backend_stubs', 'tools/codegen/gen_backend_stubs.py')
```

Comment:
@albanD I think this has come up before - right now CI isn't happy with this test because the tools directory isn't part of `sys.path` in CI. This is the best I could come up with as a workaround without modifying sys.path: I'm using `importlib` to manually import a file, given its full path.

Reply:
As discussed offline, this can move to the test folder inside `tools/` that run in a different CI job